addons/sourcemod/scripting/ne/menus.inc

```diff
@@ -336,17 +397,40 @@ public int Handler_MapSelectMenu(Menu menu, MenuAction action, int param1, int p
 				AttemptNominate(param1, "");
 				return 0;
 			}
-
+		
+			if (strcmp(map, "show_tiers", false) == 0)
+			{
+				AttemptNominate(param1, "", -1);
+				return 0;
+			}
+			
+			if (StrContains(map, "show_tier_") != -1)
+			{
+				ReplaceString(map, sizeof(map), "show_tier_", "");
+				int tier = StringToInt(map);
+				if (!tier)
+					return 0;
+					
+				AttemptNominate(param1, "", tier);
+				return 0;
+			}
+			
 			GetClientName(param1, name, MAX_NAME_LENGTH);
 
 			if (strcmp(map, "show_nominated", false) == 0)
 			{
 				char sNominated[PLATFORM_MAX_PATH];
 				if (GetNominationByOwner(param1, sNominated))
 				{
+					char mapDisplay[PLATFORM_MAX_PATH];
+					if (GetMapTierName(sNominated, mapDisplay, sizeof(mapDisplay)))
+						Format(mapDisplay, sizeof(mapDisplay), "%s %t", sNominated, "MapTierDisplay", mapDisplay);
```

Comment:
I don't think chat messages and server logs should display tiers. At most, you could have a cvar that dictates if chat should display tiers. Otherwise, I see no need for tiers being stored in logs.